### PR TITLE
Blogpost411

### PR DESCRIPTION
**Pull request by JohnMarksRH**
Fixes issue # (delete if not relevant)

Changes proposed in this pull request:

- This is a new blog post about the 4.1.1 release

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): @jmarks

This pull request needs review by: @mykaul @ykaul

### CODE REVIEW

File: source/blog/2017-04-05-say-hello-to-ovirt-4.1.1.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+---
+
+On March 22, the oVirt project released version 4.1.1, available for Red Hat Enterprise Linux 7.3, CentOS Linux 7.3, or similar.
+
+oVirt is the open source virtualization solution that provides an awesome KVM management interface for multi-node virtualization. This maintenance version is super stable and there are some nice new features.
+
+So what's new in oVirt 4.1.1?
+
+
+READMORE
```
Comment (JohnMarksRH) [inline source/blog/2017-04-05-say-hello-to-ovirt-4.1.1.md]: @mykaul @bproffitt @eedri 
Both files form part of the blog post. The top above is the snippet that you see on the main blog page, When you click READMORE you see the rest of the post, below. Brian, Eyal - What I did was simply duplicate a blog post and change the text. The outcome is not displayed in middleman. Is this the right way to do it?

Comment (bproffitt) [inline source/blog/2017-04-05-say-hello-to-ovirt-4.1.1.md]: This is correct. The READMORE has to be there.

**Comment by JohnMarksRH** [comment]
@bproffitt @eedri @mykaul Why can't we see the new post live?

**Comment by duck-rh** [comment]
So, everything is fine on the builder and publishing side. What is wrong is that this PR was not checked _thoroughly_.

In fact, you're not allowed to use multiple filename extensions because Middleman interprets it as multiple transformations (like '.md' means Markdown to HTML). So you can use dashes or underscores to encode the version ("4.1.1") in the filename.

So I would advise two improvements:

- make this extension pitfall be documented (because I guess it's not)

- suggest **strongly** that contributors and also reviewers use the ./run-server.sh script to test the result (this uses the live server so you do not need to build the whole site)

**Comment by JohnMarksRH** [comment]
@duck-rh @bproffitt I see. I have renamed the filename. But there is some kind of new clash now. There doesn't seem to be way to solve this but to delete the branch and start anew. Any suggestions?

**Comment by mykaul** [comment]
@JohnMarksRH 'git mv' didn't work?

**Pull request merged**